### ISSUE TITLE
GuardClause false positive

### ISSUE THREAD

**Issue by strongriley** [rubocop/rubocop#1372]
I have the following code:

``` ruby
  def verify_sidekiq!
    monitor = Sidekiq::Queue.new
    if monitor.size > MAX_QUEUE_SIZE
      fail "Sidekiq queue size exceeds max"
    end
  end
```

This caused a `Style/GuardClause` cop to raise.

However, when the code is instead written as below, the cop no longer fires:

``` ruby
  def verify_sidekiq!
    monitor = Sidekiq::Queue.new
    fail "Sidekiq queue size exceeds max" if monitor.size > MAX_QUEUE_SIZE
  end
```

I rewrote it to meet the line max length requirement. I would not expect this new cop to fire.

``` bash
rubocop -V
0.26.0 (using Parser 2.2.0.pre.4, running on ruby 1.9.3 x86_64-darwin13.3.0)
```
